client/src/app/pages/applications/components/application-form/application-form.tsx

```diff
@@ -316,8 +316,8 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({
     onSaved(response);
   };
 
-  const onCreateUpdateApplicationError = (error: AxiosError) => {
-    setAxiosError(error);
+  const onCreateUpdateApplicationError = (error: unknown) => {
+    if (error instanceof Error) setError(error);
```

Comment:
What is going on here?

Reply:
I restored AxiosError for now until we decide cleary on how we're going to handle/filter fetch errors.